### DISCUSSION TITLE
null-ls checkhealth showing not executable

### DISCUSSION THREAD

**Discussion by Kim-Robin** [LazyVim/LazyVim#491] [Q&A] [answered]
When I run :checkhealth, I see ERROR on null-ls saying

ERROR: fish_indent: the command "fish_indent" is not executable.
ERROR: fish: the command "fish" is not executable

Any idea what could cause these error? I don't think I have installed any fish shell related thing.

**Comment by folke** [comment, marked as answer]
don't worry about it. I've added this by default for users (like myself) that use the fish shell. But if you never open a fish script, then you don't need that, so all ok :)
